api/src/pcapi/core/mails/transactional/finance_incidents/finance_incident_notification.py

```diff
@@ -41,6 +53,57 @@ def send_finance_incident_emails(finance_incident: finance_models.FinanceInciden
         )
 
         mails.send(
-            recipients=[venue.current_reimbursement_point.bookingEmail],
+            recipients=[booking_email],
             data=data,
         )
+
+
+def send_commercial_gesture_email(finance_incident: finance_models.FinanceIncident) -> None:
+    venue = finance_incident.venue
+
+    if not FeatureToggle.WIP_ENABLE_NEW_BANK_DETAILS_JOURNEY.is_active():
+        if not venue.current_reimbursement_point or not venue.current_reimbursement_point.bookingEmail:
+            return
+        booking_email = venue.current_reimbursement_point.bookingEmail
+    else:
+        if (
+            not venue.current_bank_account_link
+            or not venue.current_bank_account_link.bankAccount
+            or not venue.bookingEmail
+        ):
+            return
+        booking_email = venue.bookingEmail
+
+    offers_incidents: dict[tuple, list[finance_models.BookingFinanceIncident]] = {}
+    for booking_finance_incident in finance_incident.booking_finance_incidents:
+        booking = booking_finance_incident.booking or booking_finance_incident.collectiveBooking
+        if isinstance(booking, bookings_models.Booking):
+            offer = booking.stock.offer
+        else:
+            offer = booking.collectiveStock.collectiveOffer
+
+        if (offer.id, offer.name) not in offers_incidents:
+            offers_incidents[(offer.id, offer.name)] = []
+        offers_incidents[(offer.id, offer.name)].append(booking_finance_incident)
+
+    template = TransactionalEmail.COMMERCIAL_GESTURE_REIMBURSEMENT
+    for offer_id, offer_name in offers_incidents:
+        offer_booking_incidents = offers_incidents[(offer_id, offer_name)]
+        data = mails_models.TransactionalEmailData(
+            template=template.value,
+            params={
+                "OFFER_NAME": offer_name,
+                "VENUE_NAME": venue.common_name,
+                "MONTANT_REMBOURSEMENT": finance_utils.to_euros(
+                    sum(booking_incident.newTotalAmount for booking_incident in offer_booking_incidents)
+                ),
+                "TOKEN_LIST": ", ".join(
+                    [
+                        (booking_incident.booking or booking_incident.collectiveBooking).token
+                        for booking_incident in offer_booking_incidents
+                    ]
```

Comment:
Les réservations collectives n'ont pas de token.

```suggestion
                "TOKEN_LIST": ", ".join(
                    [
                        booking_incident.booking.token
                        for booking_incident in offer_booking_incidents
                        if booking_incident.booking
                    ]
```

... mais attention, la variable `TOKEN_LIST` passée au template pourrait alors être vide si seulement des réservations collectives sont concernées. Est-ce que le template sait gérer cela ?